example_test.go

```diff
@@ -39,11 +39,11 @@ func Example() {
 	// # TYPE http_request_seconds histogram
 	// # UNIT http_request_seconds seconds
 	// # HELP http_request_seconds A histogram example
-	// http_request_seconds_bucket{status="200",le="0.005"} 0
-	// http_request_seconds_bucket{status="200",le="0.01"} 0
-	// http_request_seconds_bucket{status="200",le="0.05"} 0
-	// http_request_seconds_bucket{status="200",le="0.1"} 0
-	// http_request_seconds_bucket{status="200",le="0.5"} 0
+	// http_request_seconds_bucket{status="200",le="0.005"} 1
+	// http_request_seconds_bucket{status="200",le="0.01"} 1
+	// http_request_seconds_bucket{status="200",le="0.05"} 1
+	// http_request_seconds_bucket{status="200",le="0.1"} 1
```

Comment:
it's the other way round.  the bucket with `le="0.1"` (i.e. `<= 0.1`) should NOT contain the 0.56 observation but the buckets `le="1"`, `le="5"`, `le="10"` etc should contain it.

as the doc says:

> The +Inf bucket counts all requests.

Reply:
:facepalm: 

Will fix.